examples/preprocessing/plot_transformed_target.py

```diff
@@ -84,7 +75,8 @@ def exp_transform(x):
 # non-linearity, the model trained will not be precise during the
 # prediction. Subsequently, a logarithmic function is used to linearize the
 # targets, allowing better prediction even with a similar linear model as
-# reported by the median absolute error (MAE).
+# reported by the median absolute error (MAE). The `numpy` functions `expm1`
```

Comment:
I will revert this change. I don't think this is very important in the context of the example. Other than this LGTM.

By the way the Circle failures are due to the mldata website which seems to have some glitchs from time to time these days.